### ISSUE TITLE
Only initialize Hystrix once per runtime

### ISSUE THREAD

**Issue by rhusar** [smallrye/smallrye-fault-tolerance#136]
```
18:44:03,986 ERROR [org.jboss.threads.errors] (ServerService Thread Pool -- 78) Thread Thread[ServerService Thread Pool -- 78,5,ServerService ThreadGroup] threw an uncaught exception: 
org.jboss.weld.exceptions.WeldException: WELD-000049: Unable to invoke void io.smallrye.faulttolerance.HystrixInitializer.onStartup() on io.smallrye.faulttolerance.HystrixInitializer@46ddadcf
```

**Comment by rhusar** [comment]
The rationale is initial support for multiple deployments in the runtime. The spec does not say that this is not possible, while SR implementation does not take that into account.

**Comment by Ladicek** [comment]
I think that makes sense, sure. Out of curiosity, did you check it with multiple deployments, possibly being deployed and undeployed concurrently? :-)

**Comment by rhusar** [comment]
@Ladicek Yes, I have and I did not see any issues, but it was really a smoke test. Even if we did need changes, most likely those would be needed in Hystrix. And of course, I would like to avoid a jboss fork of it as much as possible.

Just to note, I don't think we will fully sign up for supporting multiple deployments at the moment, but  I don't see reason why not to fix obviously incorrect assumptions.
